.travis.yml

```diff
@@ -1,9 +1,23 @@
 language: vim
+rvm:
+  - 1.8.7 # Use stock
+  - 1.9.2 # Build latest
```

Comment:
Why do this? Wouldn't it be possible to install everything in one go, and then run two test commands separately?